torch/backends/mkl/__init__.py

```diff
@@ -1,4 +1,5 @@
 import torch
+from torch._C import _verbose
```

Comment:
I thought we were going to do `import torch._C._verbose as _verbose` inside of `./torch/_C/__init__.pyi.in`. I think that's cleaner since it should make both `torch._C._verbose` and `from torch._C import _verbose` typecheck.

Reply:
oh i see, let me try it

Reply:
it works with `from . import _verbose`!